maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/FeatureDecisionServiceTest.java

```diff
@@ -134,7 +137,7 @@ void givenUserDoesNotHaveFeatureToggle_whenIsFeatureEnabledIsInvoked_thenReturnF
                 .build();
 
         FeatureDecisionService featureDecisionService = new FeatureDecisionService(maatCourtDataService);
-        boolean result = featureDecisionService.isFeatureEnabled(request, FeatureToggle.MAAT_POST_ASSESSMENT_PROCESSING, FeatureToggleAction.UPDATE);
+        boolean result = featureDecisionService.isMaatPostAssessmentProcessingEnabled(request);
```

Comment:
I think the two tests you've added here are redundant as they're covered in the main scenarios.